drivers/README.md

```diff
@@ -1,8 +1,34 @@
 # SCOS Sensor Drivers Directory
 
-This directory is mounted as a Docker volume to `/drivers` in the SCOS sensor Docker
+This directory is mounted as a Docker volume to `/drivers` in the scos-sensor Docker
 container.
 
-Some sensors/SDRs require drivers which cannot be packaged with their respective SCOS
-plugins. Those drivers can be placed here manually, allowing them to be referenced from
-within the SCOS sensor Docker container.
+Some signal analyzers require drivers which cannot be packaged with their respective
+SCOS plugins. Those drivers can be placed here manually, allowing them to be referenced
+from within the scos-sensor Docker container.
+
+A json file can be used to copy files in this directory to a required destination in
+the Docker container.
+
+Below is a sample json file. The `"source_path"` must be relative to the `drivers`
+directory. The `"dest_path"` can be anywhere in the Docker container.
```

Comment:
I would add a note that the directory will be created if it does not exist and indicate whether or not the parent directory must already exist or not. See question in utils.py for context.